csfieldguide/templates/interactives/lzw-compression.html

```diff
@@ -0,0 +1,45 @@
+{% extends interactive_mode_template %}
+
+{% load i18n %}
+{% load static %}
+
+{% block html %}
+<div id="container">
```

Comment:
I think we need to add either some row and column classes or some padding and margins to stop the content from riding on the edge of the window.

![image](https://user-images.githubusercontent.com/16066822/56999161-2a7bc680-6c02-11e9-9fa6-70807cd552f9.png)